apps/backend/src/libs/modules/controller/base-controller.module.ts

```diff
@@ -23,13 +25,29 @@ class BaseController implements Controller {
 		this.routes = [];
 	}
 
-	private async mapHandler(
-		handler: APIHandler,
-		request: FastifyRequest,
-		reply: FastifyReply,
-	): Promise<void> {
+	private async mapHandler(options: {
+		handler: APIHandler;
+		reply: FastifyReply;
+		request: FastifyRequest;
+		validation?: ControllerRouteParameters["validation"];
+	}): Promise<void> {
+		const { handler, reply, request, validation } = options;
```

Comment:
This should already be done by Fastify when we give validation schema. Try to find other solutions and share with everyone in chat